source/plugin/offline-userplayer-data.rst

```diff
@@ -0,0 +1,71 @@
+=======================================
+Accessing data of offline Users/Players 
+=======================================
+
+It may be necessary for plugins to access player data even when the player is offline.
+Since ``Sponge.getServer().getPlayer()`` does only return Players who are online one needs to find another solution.
```

Comment:
This is sort-of an awkward sentence. `Sponge.getServer().getPlayer()` will only ever return one `Player`. Not `Player`s.
And besides, a `Player` object only exists for online players anyway. This sentence makes it sound as if the `Sponge.getServer().getPlayer()` method is at fault in that it only returns online players.

A better sentence would be "Since `Player` objects only ever exist for online players, we need to find another solution for accessing offline data."